net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/general.xml

```diff
@@ -11,6 +11,12 @@
         <type>checkbox</type>
         <help>This allows you to dynamically assign VLANs on your physical switch ports.</help>
     </field>
+    <field>
+        <id>general.ldap_enabled</id>
+        <label>Enable LDAP</label>
+        <type>checkbox</type>
+        <help>This allows you to bind to an external LDAP server, use configuration in submenu LDAP.</help>
```

Comment:
Remove second half of the text, it embeds information that can and will change long-term.